run_test.sh

```diff
@@ -434,13 +441,82 @@ dkms_test.ko${mod_compression_ext}:
    - Use the dkms install command to reinstall any previous module version.
 Deleting module dkms_test-1.0 completely from the DKMS tree.
 EOF
-run_with_expected_output dkms_status_grep_dkms_test << EOF
+run_with_expected_output dkms_status_grep_dkms_module 'dkms_test' << EOF
 EOF
 
 echo 'Removing /usr/src/dkms_test-1.0'
 rm -r /usr/src/dkms_test-1.0
 
-echo 'Checking that the environment is clean'
+echo 'Checking that the environment is clean again'
+clean_dkms_env
+
+echo 'Running autoinstall error testing'
+
+echo 'Adding failing test module by directory'
+run_with_expected_output dkms add test/dkms_failing_test-1.0 << EOF
+Creating symlink /var/lib/dkms/dkms_failing_test/1.0/source -> /usr/src/dkms_failing_test-1.0
+EOF
+echo 'Running autoinstall with failing test module (expected error)'
+run_with_expected_error 11 dkms autoinstall << EOF
+Sign command: /usr/lib/linux-kbuild-5.10/scripts/sign-file
+Signing key: /var/lib/dkms/mok.key
+Public certificate (MOK): /var/lib/dkms/mok.pub
+
+Building module:
+Cleaning build area...(bad exit status: 2)
+make -j16 KERNELRELEASE=${KERNEL_VER} all...(bad exit status: 2)
```

Comment:
`make -j$(nproc) ...` as seen elsewhere though the file.

Reply:
Done.